src/docs/running/wrapping-up.md

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
For some reason, a lot of the pages/sub-headers on this page are loading extremely slowly on my local instance, and gets stuck on `Preparing requested page` for quite some time.  It doesn't seem to happen with any of the other pages, but I'm not sure what the bottleneck could be. I'll see if i can reliably reproduce it.

Reply:
Interestingly, some folks have reported this being a unique issue with M1 macbooks - wonder if that could be the culprit? https://stackoverflow.com/questions/66681392/gatsby-error-when-creating-a-new-route-preparing-requested-page